packages/form-js-playground/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "all": "run-s lint test build",
     "build": "run-p bundle generate-types",
-    "bundle": "rollup -c --failAfterWarnings",
```

Comment:
As you already mentioned, we should not remove this, but rather investigate why it breaks.